server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -177,6 +185,7 @@ private static String normalizePath(String path) {
             Author author,
             CommitMessageDto commitMessage,
             @RequestConverter(ChangesRequestConverter.class) Iterable<Change<?>> changes) {
+        checkMirrorLocalRepo(repository, changes);
```

Comment:
Don't we also need to deal with Thrift API?

Reply:
It's time to remove Thrift API? 😆 
Thanks let me add this there too.

Reply:
> It's time to remove Thrift API? 😆

🙈